packages/babel-plugin-minify-mangle-names/src/index.js

```diff
@@ -480,7 +481,9 @@ module.exports = babel => {
           } else {
             throw new Error(
               `Unexpected Rename Error: ` +
-                `Trying to replace "${node.name}": from "${oldName}" to "${newName}". ` +
+                `Trying to replace "${
+                  node.name
```

Comment:
maybe update prettier that's used in your editor? Also, is `yarn fix` changing this too ?

Reply:
yarn fix doesnt change this but changes some other things.. I have updated the editor prettier.. something weird.